nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -6,6 +6,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 ## Highlights {#sec-release-21.11-highlights}
 
+- `iptables` now use `nf_tables` backend.
```

Comment:
I suppose this would sound better if either `use` would be replaced with `uses`, or alternatively, `the` would be inserted after `use`.